src/assets/scripts/client/scope/RadarTargetModel.js

```diff
@@ -172,8 +172,8 @@ export default class RadarTargetModel {
         this._theme = theme;
 
         this._init(aircraftModel)
-            ._initializeScratchPad()
-            .enable();
+            .enable()
+            .setDefaultScratchpad(aircraftModel);
```

Comment:
enable should be done last, only when the thing is fully ready-- so let's reverse these.

Reply:
More accurately (as discussed), we should just do the .setDefaultScratchpad() from _within_ `this._init()`.

That would leave _this_ line as:
```
this._init(aircraftModel)
    .enable();
```